rest-api/participant_enums.py

```diff
@@ -49,8 +50,8 @@ class OrderStatus(messages.Enum):
 # in questionnaire responses.
 class Race(messages.Enum):
```

Comment:
To check my understanding, we would need to backfill because these enum conversions are happening as PTC calls into the RDR API with questionnaire responses?

Reply:
That is correct.